### PR TITLE
pdctl: get region with key

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @huachaohuang @siddontang @overvenus

### CODE REVIEW

File: pdctl/command/region_command.go [hunk 1]
```diff
@@ -28,13 +30,19 @@ var (
 	regionPrefix  = "pd/api/v1/region/%s"
 )
 
+type regionInfos struct {
+	Regions *metapb.Region `json:"regions"`
```
Comment (huachaohuang) [inline pdctl/command/region_command.go]: s/regions/region

File: pdctl/command/region_command.go [hunk 2]
```diff
@@ -62,3 +70,42 @@ func showRegionCommandFunc(cmd *cobra.Command, args []string) {
...
+		fmt.Println(cmd.UsageString())
+		return
+	}
+	key := args[0]
+	client, err := getClient()
+	if err != nil {
+		fmt.Println("Error: ", err)
+		return
+	}
+	regions, peers, err := client.GetRegion([]byte(key))
```
Comment (huachaohuang) [inline pdctl/command/region_command.go]: s/regions/region
s/peers/leader

File: pdctl/command/region_command.go [hunk 3]
```diff
@@ -28,13 +30,19 @@ var (
 	regionPrefix  = "pd/api/v1/region/%s"
 )
 
+type regionInfos struct {
```
Comment (huachaohuang) [inline pdctl/command/region_command.go]: s/regionInfos/regionInfo

**Comment by huachaohuang** [comment]
Please update the related document after this is merged.

**Comment by huachaohuang** [comment]
PTAL @overvenus

**Comment by iamxy** [comment]
LGTM

**Comment by siddontang** [comment]
should support different key format, like protobuf escape output format

**Comment by huachaohuang** [comment]
Do you mean format like this "t\200\000\000\000\000\000\000\377\035_r\200\000\000\000\000\377#\277[\000\000\000\000\000\372"?

**Pull request merged**